generate.py

```diff
@@ -0,0 +1,50 @@
+from random import choice
+
+from faker import Faker
+
+from pysize import get_size
+
+
+def main():
+    """Create  a list of 200000 dicts with random data."""
+    fake = Faker()
+    keys = ['first_name', 'last_name', 'address', 'email', 'sex', 'state',
+            'city', 'country', 'phone_number']
+    data_list = [{i: choice_data(i, fake) for i in keys} for i in
+                 range(200000)]
+    return data_list
+
+
+def choice_data(data_line, fake):
+    """Generate random data."""
+    if data_line == 'sex':
+        data = choice(['man', 'woman'])
+    else:
+        data = eval('fake.' + data_line + '()')
+    return data
+
+
+def choise_job(fake, i, y):
+    """Add random job to new list."""
+    if y == 'job_position':
+        data = fake.job()
+    else:
+        data = i.get(y)
+    return data
+
+
+def job(data_list):
+    """Create a list of dicts with random job position."""
+    fake = Faker()
+    keys = ['first_name', 'last_name', 'job_position']
+    result = [{y: choise_job(fake, i, y) for y in keys} for i in data_list]
+    return result
+
+
+if __name__ == '__main__':
```

Comment:
Now, let's move this to `test_generator.py`

Reply:
Sorry? You want I to create tests for the code? Right?

Reply:
I want you to call it from a test method, yes. just put it in a simple function starting with `test_` and then, in the end, use smth like `assert actual_size == expected_size`.